custom/enikshay/management/commands/utils.py

```diff
@@ -80,9 +85,9 @@ def handle(self, domain, log_file_name, case_ids, **options):
 
             print("Finished at {}".format(datetime.datetime.utcnow()))
 
-    def is_valid_case(self, domain, case):
+    def is_valid_case(self, domain, case_id):
         try:
-            person_case_id = get_person_case(domain, case.case_id)
+            person_case_id = get_person_case(domain, case_id)
```

Comment:
this method should be named `get_person_case_id`.

Reply:
good observation